src/main/java/com/gtnewhorizons/modularui/common/widget/ChangeableWidget.java

```diff
@@ -74,18 +90,21 @@ private void notifyChange(boolean sync) {
 
     private void initQueuedChild() {
         if (this.queuedChild != null) {
-            IWidgetParent.forEachByLayer(this.queuedChild, widget -> {
+            final Consumer<Widget> initChildrenWrapper = widget -> {
                 if (widget instanceof IWidgetParent) {
                     ((IWidgetParent) widget).initChildren();
                 }
-            });
+            };
+            initChildrenWrapper.accept(this.queuedChild);
```

Comment:
Isn't this call redundant since `IWidgetParent#forEachByLayer` applies consumer to the top parent (`this.queuedChild` here) as well?

Reply:
Look here:
https://github.com/GTNewHorizons/KubaTech/pull/58/files#diff-ffb97b417509e89b3abbbeef466819902ee4e7efb0dfe313e31959c6708ed770R798-R799
Without this call the scrollbar is invisible, If the top element is IParentWidget then forEachByLayer will execute for all childrens without the widget itself

Reply:
Oops my bad